umbraco/Umbraco-CMS
# V14: Fix entity type container repository get

# Notes
- Fixed a bug, where the `data-type/folde/{id}`, would throw errors, if you provided a non-folder entity key. (For example, using a data-type key, not a data-type-folder key)


# How to test
- Go to settings, and click one of you datatypes, copy the key of the datatype.
- Call the`data-type/folde/{id}` endpoint with the data type key, this should now return a 404, not a 500
